src/components/ProductCard/ProductCard.tsx

```diff
@@ -66,6 +71,8 @@ export default function ProductCard(props: ProductCardProps) {
   const { formatPrice } = state.formatters;
   const onClick = useOnProductCardClick(client, state.callbacks.onProductCardClick);
 
+  const cnstrcData = getProductCardCnstrcDataAttributes(item, productInfo);
```

Comment:
Should only pass `productInfo`

```suggestion
  const cnstrcData = getProductCardCnstrcDataAttributes(productInfo);
```